### PR TITLE
Update system-variables.md

### PR DESCRIPTION
**Pull request by joey-yez**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -1581,6 +1581,7 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
 - 这个变量用来设置 scan 操作的并发度。
 - AP 类应用适合较大的值，TP 类应用适合较小的值。对于 AP 类应用，最大值建议不要超过所有 TiKV 节点的 CPU 核数。
 - 若表的分区较多可以适当调小该参数（取决于扫描数据量的大小以及扫描频率），避免 TiKV 内存溢出 (OOM)。
+- 对于只有 LIMIT 的简单查询，在 scan 操作被下推到 TiKV 时，为了提升执行效率，如果 LIMIT 行数小于 100000，该变量将按照 1 进行处理。
```
Comment (joey-yez) [inline system-variables.md:1584]:
```suggestion
- 对于只有 LIMIT 的简单查询，在 scan 操作被下推到 TiKV 时，为了提升执行效率，如果 LIMIT 行数小于 100000，该变量将按照 1 进行处理。查询语句 `SELECT MAX(col), MIN(col) FROM ...` 也会改写为 `SELECT col FROM ... ORDER BY col LIMIT 1`，该变量也将按照 1 进行处理。
```

File: system-variables.md [hunk 2]
```diff
@@ -1581,6 +1581,7 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
 - 这个变量用来设置 scan 操作的并发度。
 - AP 类应用适合较大的值，TP 类应用适合较小的值。对于 AP 类应用，最大值建议不要超过所有 TiKV 节点的 CPU 核数。
 - 若表的分区较多可以适当调小该参数（取决于扫描数据量的大小以及扫描频率），避免 TiKV 内存溢出 (OOM)。
+- 对于只有 LIMIT 的简单查询，在 scan 操作被下推到 TiKV 时，为了提升执行效率，如果 LIMIT 行数小于 100000，该变量将按照 1 进行处理。查询语句 `SELECT MAX/MIN(col) FROM ...` 也会改写为 `SELECT col FROM ... ORDER BY col LIMIT 1`，该变量也将按照 1 进行处理。
```
Comment (joey-yez) [inline system-variables.md:1584]:
```suggestion
- 对于只有 LIMIT 的简单查询，在 scan 操作被下推到 TiKV 时，为了提升执行效率，如果 LIMIT 行数小于 100000，该变量将按照 1 进行处理。查询语句 `SELECT MAX/MIN(col) FROM ...` 在 col 列包含索引、具有有序性时，也会改写为 `SELECT col FROM ... ORDER BY col LIMIT 1`，该变量也将按照 1 进行处理。
```

**Review by XuHuaiyu — APPROVED** [review]

File: system-variables.md [hunk 3]
```diff
@@ -1581,6 +1581,7 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
 - 这个变量用来设置 scan 操作的并发度。
 - AP 类应用适合较大的值，TP 类应用适合较小的值。对于 AP 类应用，最大值建议不要超过所有 TiKV 节点的 CPU 核数。
 - 若表的分区较多可以适当调小该参数（取决于扫描数据量的大小以及扫描频率），避免 TiKV 内存溢出 (OOM)。
+- 对于只有 LIMIT 的简单查询，在 scan 操作被下推到 TiKV 时，为了提升执行效率，如果 LIMIT 行数小于 100000，该变量将按照 1 进行处理。查询语句 `SELECT MAX/MIN(col) FROM ...` 在 col 列有索引、有序性满足要求时，会改写为 `SELECT col FROM ... LIMIT 1`，该变量也将按照 1 进行处理。
```
Comment (qiancai) [inline system-variables.md:1584]:
```suggestion
- 对于仅包含 `LIMIT` 子句的简单查询，如果 `LIMIT` 行数小于 100000，该查询的 scan 操作被下推到 TiKV 时，会将该变量的值视为 `1` 进行处理，以提升执行效率。
- 对于查询语句 `SELECT MAX/MIN(col) FROM ...`，如果 `col` 列有索引且满足有序性要求，TiDB 会将该查询改写为 `SELECT col FROM ... LIMIT 1` 进行处理，该变量的值也将视为 `1` 进行处理。
```

File: system-variables.md [hunk 4]
```diff
@@ -1581,6 +1581,8 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
 - 这个变量用来设置 scan 操作的并发度。
 - AP 类应用适合较大的值，TP 类应用适合较小的值。对于 AP 类应用，最大值建议不要超过所有 TiKV 节点的 CPU 核数。
 - 若表的分区较多可以适当调小该参数（取决于扫描数据量的大小以及扫描频率），避免 TiKV 内存溢出 (OOM)。
+- 对于仅包含 `LIMIT` 子句的简单查询，如果 `LIMIT` 行数小于 100000，该查询的 scan 操作被下推到 TiKV 时，会将该变量的值视为 `1` 进行处理，以提升执行效率。
+- 对于查询语句 `SELECT MAX/MIN(col) FROM ...`，如果 `col` 列有索引且满足有序性要求，TiDB 会将该查询改写为 `SELECT col FROM ... LIMIT 1` 进行处理，该变量的值也将视为 `1` 进行处理。
```
Comment (qiancai) [inline system-variables.md:1585]:
```suggestion
- 对于查询语句 `SELECT MAX/MIN(col) FROM ...`，如果 `col` 列有索引且该索引的数据是按顺序排列的，TiDB 会将该查询改写为 `SELECT col FROM ... LIMIT 1` 进行处理，该变量的值也将视为 `1` 进行处理。
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

File: system-variables.md (hunk 4 above)
Comment (joey-yez) [reply system-variables.md:1585]: 索引都是有序的，这里主要是索引的顺序和要计算的 max/min 一致。例如索引是升序的，那么计算 min(col) 就只需要取 col 的第一条数据。

**Pull request merged**